daphne/src/roles/leader.rs

```diff
@@ -98,6 +99,17 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
     /// Data type used to guide selection of a set of reports for aggregation.
     type ReportSelector;
 
+    /// Register these report ids as received by the leader. Returning all ids that have been
```

Comment:
I think we're already using the term "processed" for what you mean by "registered" here?